packages/dak-components/lib/crescat.ts

```diff
@@ -0,0 +1,97 @@
+import axios, { AxiosRequestHeaders } from 'axios'
+
+import cache from 'memory-cache';
+
+// Fetches the data from a given URL
+const cachedGet = async (url: string, headers: AxiosRequestHeaders) => {
+    const cachedResponse = cache.get(url);
+    if (cachedResponse) {
+        return cachedResponse;
+    }
+
+    const cacheTime = 1000 * 60 * 30; //Cached for 1 minutes
+    const response = await axios.get(url, {
+        headers
+    });
+
+    // Establishes the data from the URL-response
+    const data = response.data;
+    cache.put(url, data, cacheTime);
+    return data;
+}
+
+
+const getEvents = async () => {
+
+    // Fetches json payload from the Crescat booking site
+    const crescatData = await cachedGet("https://app.crescat.io/external/v1/calendar", {
+        Authorization: `Bearer ${process.env.CRESCAT_TOKEN}`
+    });
+
+    var rawShowTimes = Array();
```

Comment:
```suggestion
    var rawShowTimes = Array();
```
```suggestion
    const rawShowTimes: Show_Times[][] = [];
```
Use const and let instead of var in javscript